util/execdetails/execdetails.go

```diff
@@ -415,6 +415,42 @@ func (d ExecDetails) ToZapFields() (fields []zap.Field) {
 	return fields
 }
 
+type basicCopRuntimeStats struct {
+	BasicRuntimeStats
+	// concurrency
+	concurrency int32
```

Comment:
uint8?

Reply:
TiDB use `int` type to describe the [concurrency,](https://github.com/windtalker/tidb/blob/5e67a597ccdd8220f40d69bc601f1b664949f885/sessionctx/variable/session.go#L1826) so I choose `int32` here.